eox_nelp/pearson_vue/pipeline.py

```diff
@@ -58,9 +73,176 @@ def get_user_data(user_id, **kwargs):  # pylint: disable=unused-argument
+                            "phoneNumber": kwargs["phone_number"],
+                            "phoneCountryCode": kwargs["phone_country_code"],
+                        },
+                        "mobile": {
+                            "mobileNumber": kwargs["mobile_number"],
+                            "mobileCountryCode": kwargs["mobile_country_code"],
+                        }
+                    }
+                },
+            },
+        },
+    }
+    payload = update_xml_with_dict(PAYLOAD_CDD, payload)
+    response = api_client.import_candidate_demographics(payload)
+
+    if response.get("status", "error") != "accepted":
+        # pylint: disable=broad-exception-raised
+        raise Exception("Error trying to process import candidate demographics request.")
+
+
+def import_exam_authorization(transaccion_type="Add", **kwargs):
+    """
+    Imports exam authorization data into the Pearson VUE RTI system.
+
+    This function creates  a payload with exam  authorization details and  sends a
+    request to the Pearson VUE RTI service to import this information. The payload
+    includes details such as the candidate ID, exam  series code, and  eligibility
+    appointment dates. If the  import request is not accepted, the function raises
+    an exception.
+
+    Args:
+        transaccion_type (str): The type of transaction for the authorization (default is "Add").
+        **kwargs: A dictionary containing the following key-value pairs:
+            - anonymous_user_id (str): An anonymized identifier for the user.
+
+    Raises:
+        Exception: If the Pearson VUE RTI service does not accept the import request.
+    """
+    api_client = PearsonRTIApiClient()
+    payload = {
+        "soapenv:Envelope": {
+            "soapenv:Header": {
+                "wsse:Security": {
+                    "wsse:UsernameToken": {
+                        "wsse:Username": getattr(settings, "PEARSON_RTI_WSDL_USERNAME"),
+                        "wsse:Password": {
+                            "#text": getattr(settings, "PEARSON_RTI_WSDL_PASSWORD")
+                        },
+                    },
+                },
+            },
+            "soapenv:Body": {
+                "sch:eadRequest": {
+                    "@clientID": getattr(settings, "PEARSON_RTI_WSDL_CLIENT_ID"),
+                    "@authorizationTransactionType": transaccion_type,
+                    "clientCandidateID": f'NELC{kwargs["anonymous_user_id"]}',
+                    "examAuthorizationCount": kwargs["exam_authorization_count"],
+                    "examSeriesCode": kwargs["exam_series_code"],
+                    "eligibilityApptDateFirst": timezone.now().strftime("%Y/%m/%d %H:%M:%S"),
+                    "eligibilityApptDateLast": (
+                        timezone.now() + timezone.timedelta(days=365)
+                    ).strftime("%Y/%m/%d %H:%M:%S"),
+                    "lastUpdate": timezone.now().strftime("%Y/%m/%d %H:%M:%S GMT"),
```

Comment:
I thing here would be UTC OR GMT, but now I know that the server timezone is UTC.
![image](https://github.com/eduNEXT/eox-nelp/assets/51926076/1a94a546-eb81-4338-8261-6225d8434dbe)

Reply:
The service requires the GMT word at the end, if we don't change the serve timezone we shouldn't have any problem